packages/block-library/src/post-author/index.php

```diff
@@ -35,12 +35,12 @@ function render_block_core_post_author( $attributes, $content, $block ) {
 		isset( $attributes['textAlign'] ) ? array( 'has-text-align-' . $attributes['textAlign'] ) : array()
 	);
 
-	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => implode( ' ', $classes ) ) );
+	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => esc_attr( implode( ' ', $classes ) ) ) );
```

Comment:
Same as above. Values in `get_block_wrapper_attributes` should not be escaped because the function already escapes them.

Reply:
I agree here that this isn't necessary.